packages/superset-ui-chart-controls/src/shared-controls/index.tsx

```diff
@@ -429,6 +429,16 @@ const y_axis_format: SharedControlConfig<'SelectControl'> = {
   },
 };
 
+const x_axis_time_format: SharedControlConfig<'SelectControl'> = {
+  type: 'SelectControl',
+  freeForm: true,
+  label: t('X Axis Format'),
+  renderTrigger: true,
+  default: 'SMART_NUMBER',
+  choices: D3_FORMAT_OPTIONS,
```

Comment:
The name of this variable says time format, but these choices and the default value are for number formats?